### PR TITLE
Complete CRUD routes for Cassandra Db and PostgreSQL

### PR DESCRIPTION
**Pull request by velezjose**
This PR accomplishes establishing the CRUD interface from the **cassandra/index.js** & **postgresql/index.js** files to their respective DBMSs.

1. The **cassandra/index.js** interface was refactored from callback-based to promise-based.
2. The **postgresql/index.js** interface was built entirely to be promise-based, this includes the helper functions to generate the query strings, found in **postgresql/queryStringHelpers.js**.
3. The **data/helpers.js** exports were changed to create the **dataNoId.csv** file which does not include the `id` header or the value. This `.csv` file was used to re-seed the PostgreSQL database, taking advantage of the `SERIAL` data type option for a `PRIMARY KEY`, which auto-incremements that `id`.

Btw, the `schema.cql` and `schema.sql` files are only there for reference, since all database seeding was done in the databases' respective shells.

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -25,3 +25,4 @@ build/
```
Comment (TylerNguyen21) [inline .gitignore:25]: everything looks good for the most part.  Consider adding your package-lock.json and your csv files in your git ignore

**Review by TylerNguyen21 — APPROVED** [review]
Just read my comment from above

File: .gitignore (hunk 1 above)
Comment (velezjose) [reply .gitignore:25]: Thanks. I added these example*.csv files so that developers can know the shape of the data.

Comment (velezjose) [reply .gitignore:25]: Forgot to mention, I researched a bit if I should include the **package-lock.json** file, and it should be included and pushed to the remote Github repo according to several StackOverFlow posts.
(_https://stackoverflow.com/questions/44206782/do-i-commit-the-package-lock-json-file-created-by-npm-5_)

**Pull request merged**